### ISSUE TITLE
Annotations lists are not tagged as semantic list

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#2729]
The annotation panels display annotations as a list, but they are not semantically identified. 

![screenshot with code: <div class="annotations_line" style="border-left: 4px solid rgb(212, 196, 251);"><div class="annnotation_container"><button class="annotation_name R2_CSS_CLASS__FORCE_NO_FOCUS_OUTLINE" aria-label="Go to ... &quot;Solid background&quot;" id="9c7f444f-f52a-4616-aebc-8613c5607a1c"><p>Solid background</p></button><p><span></span><br></p></div><div class="annotation_edit"><div><div><svg aria-hidden="true" viewBox="0 0 14 14"><use xlink:href="#calendar-icon"></use></svg><p>19/12/2024</p></div><div><svg aria-hidden="true" viewBox="0 0 14 13"><use xlink:href="#bookOpen-icon"></use></svg><p>59% </p></div></div><div class="annotation_actions_buttons"><button id="annotation_card-9c7f444f-f52a-4616-aebc-8613c5607a1c_edit_button" title="Edit"><svg aria-hidden="true" viewBox="0 0 11 11"><use xlink:href="#pen-icon"></use></svg></button><button title="Delete" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-:r16v:" data-state="closed"><svg aria-hidden="true" viewBox="0 0 16 18"><use xlink:href="#trash-icon"></use></svg></button></div></div><div class="gauge"><div class="fill" style="width: 58.99%;"></div></div></div>](https://github.com/user-attachments/assets/a2339498-fbb9-462d-884b-c04b2500165d)

**Comment by gautierchomel** [comment]
As a consequence, screen reader users cannot know how many annotations are present nor list them.

**Comment by gautierchomel** [comment]
This also applies to Bookmarks.
